lib/drivers/animasDriver.js

```diff
@@ -1644,28 +1644,31 @@ module.exports = function (config) {
 
       }
 
-      var suspend = null;
-      var resume = null;
       for(var s in data.suspendResumeRecords) {
         var suspendresumedatum = data.suspendResumeRecords[s];
 
-        suspend = cfg.builder.makeDeviceEventSuspend()
-          .with_deviceTime(suspendresumedatum.suspendDeviceTime)
-          .with_reason({suspended: 'manual'})
-          .set('index', suspendresumedatum.index);
-        cfg.tzoUtil.fillInUTCInfo(suspend, suspendresumedatum.suspendJsDate);
-        suspend = suspend.done();
-        postrecords.push(suspend);
-
         if(suspendresumedatum.resumeJsDate > suspendresumedatum.suspendJsDate) {
+          var suspendResume = cfg.builder.makeDeviceEventSuspendResume()
+            .with_deviceTime(suspendresumedatum.suspendDeviceTime)
+            .with_reason({suspended: 'manual', resumed: 'manual'})
+            .with_duration(suspendresumedatum.resumeJsDate - suspendresumedatum.suspendJsDate)
+            .with_payload({
+              resumeDeviceTime : suspendresumedatum.resumeDeviceTime
+            })
+            .set('index', suspendresumedatum.index);
+          cfg.tzoUtil.fillInUTCInfo(suspendResume, suspendresumedatum.suspendJsDate);
+          postrecords.push(suspendResume.done());
+        }
+        else {
```

Comment:
It's the latter. I'm changing it to now always put the resume timestamp in the payload for suspends. I don't want to check specifically for 0 Jan 2007, as it may be different for the Vibe (and 0 Jan 2007 is also displayed in the UI, so it's not "wrong").